### PR TITLE
improvement(events): convert ClusterHealthValidator to continuous

### PR DESCRIPTION
**Pull request by juliayakovlev**
Convert ClusterHealthValidator event to be continuous events and subevents to
Informational.

Example:
```
2021-08-29 16:22:47.408: (ClusterHealthValidatorEvent Severity.NORMAL) period_type=begin event_id=8ef83194-fc37-4a62-ba28-bb4734ae4f69
2021-08-29 16:24:42.450: (ClusterHealthValidatorEvent Severity.CRITICAL) period_type=one-time event_id=4891e18d-a199-4aed-b920-1164e2798c16: type=NodeStatus node=longevity-10gb-3h-convert--db-node-e5ac27be-1 error=Current node 10.0.1.129. Node with 10.0.2.23 (not target node) status is DN
2021-08-29 16:26:40.718: (ClusterHealthValidatorEvent Severity.CRITICAL) period_type=one-time event_id=10756144-7fb1-4836-a5ff-c181ff0d5518: type=NodeStatus node=longevity-10gb-3h-convert--db-node-e5ac27be-2 error=Current node 10.0.3.254. Node with 10.0.2.23 (not target node) status is DN
2021-08-29 16:26:58.739: (ClusterHealthValidatorEvent Severity.WARNING) period_type=one-time event_id=60078450-6f79-4348-8092-7a2fe2214069: type=NodeStatus node=longevity-10gb-3h-convert--db-node-e5ac27be-4 message=Unable to get nodetool status from `longevity-10gb-3h-convert--db-node-e5ac27be-4': 'NoneType' object has no attribute 'stdout'
2021-08-29 16:28:18.824: (ClusterHealthValidatorEvent Severity.NORMAL) period_type=end event_id=8ef83194-fc37-4a62-ba28-bb4734ae4f69 duration=5m31s message=Cluster health check finished
```

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Review by KnifeyMoloko — APPROVED** [review]
looks good

File: sdcm/sct_events/health.py [hunk 1]
```diff
@@ -40,27 +42,35 @@ def __init__(self,
...
+            return message
+        return message
+
+
+ClusterHealthValidatorEvent.add_subevent_type("NodeStatus", mixin=ClusterHealthValidatorSubEvents)
+ClusterHealthValidatorEvent.add_subevent_type("NodePeersNulls", mixin=ClusterHealthValidatorSubEvents)
+ClusterHealthValidatorEvent.add_subevent_type("NodeSchemaVersion", mixin=ClusterHealthValidatorSubEvents)
+ClusterHealthValidatorEvent.add_subevent_type("NodesNemesis", mixin=ClusterHealthValidatorSubEvents)
+ClusterHealthValidatorEvent.add_subevent_type("MonitoringStatus", mixin=ClusterHealthValidatorSubEvents)
+ClusterHealthValidatorEvent.add_subevent_type("ScyllaCloudClusterServerDiagnostic",
```
Comment (enaydanov) [inline sdcm/sct_events/health.py:72]: It's too hacky :)

All these subtypes will be informational and continuous same time.

Comment (juliayakovlev) [reply sdcm/sct_events/health.py:72]: It's reported as informational, not continuous

**Review by enaydanov — CHANGES_REQUESTED** [review]
What do you think about having the Continuous event as a separate event? It'll simplify inheritance.

**Review by bentsi — APPROVED** [review]

**Pull request merged**